lib/elasticity/strategies/alias_index.rb

```diff
@@ -4,6 +4,10 @@ module Strategies
     # runtime changes by simply atomically updating the aliases. For example, look at the remap method
     # implementation.
     class AliasIndex
+      RETRYABLE_ERROR_SNIPPETS = [
+        "Cannot delete indices that are being snapshotted"
+      ]
```

Comment:
Beware that [constants are mutable](http://www.informit.com/articles/article.aspx?p=2251208&seqNum=4), we want to manually freeze them.